### PR TITLE
Login

### PR DESCRIPTION
**Pull request by maximiliangarmatsch**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,5 @@
 # scrapesome
-Scrape SoMe Data
+
+```bash
+$ python main.py <username> <password>
```
Comment (maximiliangarmatsch) [inline README.md:6]: was macht das? einfach ne überschrift dazu oder so reicht mir

File: insta_scrape.py [hunk 2]
```diff
@@ -0,0 +1,70 @@
+from selenium.webdriver.common.by import By
+import time
+
+waiting_seconds = 4
```
Comment (maximiliangarmatsch) [inline insta_scrape.py:4]:
```suggestion
WAITING_SECONDS = 4
```

Comment (maximiliangarmatsch) [reply insta_scrape.py:4]: und folgende...

File: insta_scrape.py [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+
+def handle_cookiebanner(driver, allow_cookies=1):
+    print('handle cookie banner..')
```
Comment (maximiliangarmatsch) [inline insta_scrape.py:12]: kann weg?

File: insta_scrape.py [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+    time.sleep(waiting_seconds)
+    button_class = '_a9--'
+    allow_cookies_button_class = '_a9_0'
+    decline_cookies_button_class = '_a9_1'
```
Comment (maximiliangarmatsch) [inline insta_scrape.py:16]: warum sind die classes so weird?

File: main.py [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+    close_saving_login_modal,
+    close_notification_modal,
+    open_follower_modal
+)
+
+if len(sys.argv) < 3:
+    print('Argument error.')
+    sys.exit()
+
+allow_cookies = 1
```
Comment (maximiliangarmatsch) [inline main.py]:
```suggestion
cookies_allowed = 1
```

File: main.py [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+
+driver.get('https://www.instagram.com/')
+
+handle_cookiebanner(driver)
+login(driver, login_username, login_password)
+close_saving_login_modal(driver)
+close_notification_modal(driver)
+
+driver.get('https://www.instagram.com/berghain_ostgut/')
+open_follower_modal(driver)
```
Comment (maximiliangarmatsch) [inline main.py]: make this a function, for example "fetchInstagramAndVisit(page)"

File: insta_scrape.py (hunk 4 above)
Comment (noxx-digital) [reply insta_scrape.py:16]: The CSS classes in instagram are completley obfuscated. These classes are CSS classes and are used further down to select the right HTML element.

File: main.py (hunk 5 above)
Comment (noxx-digital) [reply main.py]: Using past seems to be more readable. Check

File: main.py (hunk 6 above)
Comment (noxx-digital) [reply main.py]: no never, this hides just what the programm does. The programm IS this!

I will add Comments here and fetchInstagramAndVisit() is rejected.

File: README.md (hunk 1 above)
Comment (noxx-digital) [reply README.md:6]: Just a neat hint that u have to pass username and password as programm arguments. I will add some text that explains this behaviour.

File: insta_scrape.py (hunk 3 above)
Comment (noxx-digital) [reply insta_scrape.py:12]: No, Logging is imporant. While using linux system we can use bash scripts to log our executions.

```pyhton
$ python main.py <username> <password> > /var/log/some_log.log
```

If you dont want any output on Commandline use "> /dev/null 2>&1".

Comment (noxx-digital) [reply insta_scrape.py:12]: But yeah, these print lines are very minimalistic and have to be refactored at any point.

**Pull request merged**